DataFormats/MuonReco/src/classes_def.xml

```diff
@@ -177,5 +177,24 @@
   <class name="edm::PtrVector<reco::Muon>" />
   <class name="edm::Wrapper<edm::PtrVector<reco::Muon> >" />
 
+  <selection>
+    <class name="std::vector<EmulatedME0Segment>"/>
+    <class name="EmulatedME0Segment" ClassVersion="12">
+      <version ClassVersion="12" checksum="1065224607"/>
+      <version ClassVersion="11" checksum="2771665507"/>
+    </class>
+    <class name="edm::Wrapper<std::vector<EmulatedME0Segment> >"/>
+    
+    <class name="edm::Ref<std::vector<EmulatedME0Segment>,EmulatedME0Segment,edm::refhelper::FindUsingAdvance<std::vector<EmulatedME0Segment>,EmulatedME0Segment> >"/>
+    
+    <class name="std::vector<reco::ME0Muon>"/>
+    <class name="reco::ME0Muon" ClassVersion="17">
+      <version ClassVersion="17" checksum="3425786499"/>
```

Comment:
I see that 62XSLHC has ME0Muon class version 15
and EmulatedME0Segment version 11.
Was this update in the class version required?